docs/v1/P4Runtime-Spec.mdk

```diff
@@ -5591,8 +5602,9 @@ An architecture may introduce new table match types [@P4MatchTypes]. P4Runtime
 accounts for this by providing the following hooks:
 
 * The `match` field in `p4.config.v1.MatchField` (p4info.proto) is a `oneof`
-  which can be either one of the default match types (`EXACT`, `LPM`, `TERNARY`
-  or `RANGE`) or an architecture-specific match type encoded as a string.
+  which can be either one of the default match types (`EXACT`, `LPM`, `TERNARY`,
+  `RANGE`, OR `OPTIONAL`) or an architecture-specific match type encoded as a
```

Comment:
minor typo: restore capitalized "OR" back to lower case "or" here?

Reply:
Done